### PR TITLE
Feature/add favicon and alter copy

### PR DESCRIPTION
**Pull request by mckayward**
@heyjonuland - Let me know what you think of the small copy changes I'm proposing here

@SMLMRKHLMS - Not positive that I added the favicon in the right place. I couldn't build the project because I don't have a valid ssh key for some of the things that are being pulled from ideawork's private repos. If anything I was just trying to navigate around the code base a bit.

### CODE REVIEW

**Comment by mckayward** [comment]
Also, we haven't talked about the patterns we'll use for project collaboration. I like [git flow](http://nvie.com/posts/a-successful-git-branching-model/), but I am open to whatever. I thought this would be a good time to get the conversation started.

**Comment by heyjonuland** [comment]
@mckayward I'm good with the copy changes. they're great!

**Comment by mckayward** [comment]
Okay I also made the update to the copy above the form based on our slack conversation. @heyjonuland I noticed that you have been using a different logo recently, how hard is it to switch that in, @SMLMRKHLMS?

I have a suggestion for the success modal. "Thank you for your submission" feels a little generic compared to how extremely slick the rest of the site is. I'm not sure where to change it, but my digging makes me think it's part of the iws-form-component that we use. Can we update that to something like "Thanks. We'll be in touch."? (@heyjonuland feel free to critique any of the copy I have here.)

**Comment by mckayward** [comment]
@heyjonuland I know this is super nit-picky. But hey, this is potentially our livelihood we're talking here. What are your thoughts on oxford commas? I've always used them because I think that's what they taught us to use in business school. But I presume you've thought about the subject much more than me. Especially in the context of copy writing. @SMLMRKHLMS you have an opinion on this?

File: src/app/scripts/components/views/Home/Home.js [hunk 1]
```diff
@@ -36,14 +36,15 @@ class Home extends React.Component {
         <section className={ homeSection }>
           <div className={ homeContent }>
             <h2>{`Auditor-client communications just got easier.`}</h2>
-            <p>{`Narra.io is an online platform that provides an organized, efficient and secure file-sharing service designed for streamlining the auditing process and reducing fixed costs for small to mid-sized firms.`}</p>
+            <p>{`Narra is an online platform that provides a secure, efficient and organized file-sharing service designed specifically for streamlining the audit process and increasing efficiencies for small to mid-sized firms.`}</p>
+            <p>{`Spend less time tracking down audit documents and more time adding value for your clients.`}</p>
```
Comment (mckayward) [inline src/app/scripts/components/views/Home/Home.js]: @heyjonuland and @SMLMRKHLMS I added another line of copy. Too much?

**Pull request merged**